ios/Video/RCTVideo.m

```diff
@@ -869,11 +869,12 @@ - (void)setPaused:(BOOL)paused
       [[AVAudioSession sharedInstance] setCategory:AVAudioSessionCategoryAmbient error:nil];
     }
     
-    if (@available(iOS 10.0, *) && !_automaticallyWaitsToMinimizeStalling) {
-      [_player playImmediatelyAtRate:_rate];
-    } else {
-      [_player play];
-      [_player setRate:_rate];
+    if (_paused) {
```

Comment:
This if statement will never be true as we check if paused is false above. Therefor play will never be called, how can this work for you?

Reply:
I _think_ the `_paused` here is checking the previous state so it will be called if the player is playing, but the previous state was paused.